enhancements/ptp-ha.md

```diff
@@ -0,0 +1,245 @@
+following dual nic with two ptp4l configurations, but no HA on failure.
+
+```text
+        ┌─────────────────────────┐
+        │                         │
+        │                         │
+        │    Grand Master         │
+        │                         │
+        │                         │
+        └───────┬───────────┬─────┘
+                │           │
+                │           │
+                │           │
+         ┌──────┼───────────┼────┐
+         │ vDU  │           │    │
+         │ ┌────┴───┐  ┌────┴──┐ │
+         │ │        │  │       │ │
+         │ └─────┬──┘  └────┬──┘ │
+         │  NIC1 │      NIC2│    │
+         └───────┼──────────┼────┘
+                 │          │
+                 │          │
+          ┌──────┼──────────┼───┐
+          │      │          │   │
+          │      │       ┌──┘   │
+          │      │       │      │
+         ┌┼┬────┬┼┐     ┌┼┬────┬┼┐
+         ├─┘    └─┤     ├─┘    └─┤
+         │        │     │        │
+         │        │     │        │
+         │        │     │        │
+         │        │     │        │
+         │        │     │        │
+         │        │     │        │
+         │        │     │        │
+         │        │     │        │
+         │ RU#0   │     │ RU#1   │
+         └────────┘     └────────┘
+```
+
+HA on failure would require several NICs connection at the Remote Unit (RU)
+level as well, so they would be able to load balance between the two boundary
+clocks within the boundary clock.
+
+### Multiple interfaces on one NIC
+
+- Should this be covered?
+
+## Motivation
+
+### Goals
+
+- Support multiple PTP sources within the PTP operator
+- Add an additional field in the CR regarding stratum priority
+
+### Non-Goals
+
+## Proposal
+
+Would this be a new controller on the operator? Some new parameter on the
+reconcile loop? Opening this EP so we can iterate on this.
+
+### Workflow Description
+
+### API Extensions
+
+### User Stories
+
+## Design Details
+
+The linuxptp package includes ptp4l and phc2sys programs for clock
+synchronization. The PTP Operator runs these two programas into a
+linuxptp-daemon pod on each node which belongs to the cluster.
+
+### ptp4l
+
+It implements the PTP boundary clock and ordinary clock. It synchronizes the
+PTP hardware clock (PHC) from the NIC to the source clock with hardware time
+stamping and synchronizes the system clock to the source clock with *software*
+time stamping.
+
+### phc2sys
+
+phc2sys is used for hardware time stamping to synchronize the system clock to
+the PTP hardware clock on the network interface controller (NIC).
+
+### pmc
+
+It's an utility to configure ptp4l in runtime. It's also available within the
+linutptp-daemon pod.
+
+## Alternatives
+
+Using `boundary_clock_jbod` option within the profiles allows a node acting as
+a BC using two different NICs to have their PHCs synced via `phc2sys`. That
+said, this would go over the max time error (30 ns) due to the increased
```

Comment:
just clarifying this, the Max. Absolute Time Error (max|TE|) requirement of 30 ns applies to the fronthaul transport network